src/Features/Core/Portable/Shared/TestHooks/AsynchronousOperationListener.cs

```diff
@@ -23,9 +23,6 @@ internal partial class AsynchronousOperationListener : IAsynchronousOperationLis
         public AsynchronousOperationListener()
         {
             TrackActiveTokens = Debugger.IsAttached;
-
```

Comment:
this is simple change where TrackActiveTokens are assigned twice. probably someone checked in this by accident. I am reverting the change.